### ISSUE TITLE
Add support for inverting gamepad axes

### ISSUE THREAD

**Issue by 3v1n0** [AntiMicroX/antimicrox#1032]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe.

I've been using the Nintendo Switch joycon in linux, but they require some manual mapping and axis needs to be inverted when used in split mode

### Describe the solution you'd like

I would like a way to invert the direction of the axes (e.g. moving from left -> right becomes right -> left and same for vertical axis)

**Comment by pktiuk** [comment]
@3v1n0    
Why this is an issue?   
You can just map right movement to the action linked with left (for example for some games with WASD movement scheme you can map left to D instead of A)

**Comment by logicguy1** [comment]
This is an issue if using a fanatech wheel with pedals, the pedals need to be inverted to work as they do full tilt for idle position

**Comment by lestcape** [comment]
Yes, same problem with the Logitech G29 steering wheel pedals. They active a keyboard key when they are not pressed and deactivated it when pressed, causing the opposite to what we want to have. Some way to inverting the axes is needed for that case.

**Comment by lestcape** [comment]
This is a workaround for the issue:

After configured your mapping close the antimicrox app and then open the file ~/.config/antimicrox and add `~` ad the end of the axis Mappings option you want to invert. Example: `a:a2` will be `a:a2~`. Then save the file and open antimicrox again without to reconfigure anything.

More info about how to invert the axis on SDL2 setting directly:
https://github.com/mdqinc/SDL_GameControllerDB/issues/162

In my case, this will be:

```
[Mappings]
0300f04c6d0400004fc2000011010000113349743="0300f04c6d0400004fc2000011010000113349743,Logitech G29 Driving Force Racing Wheel,platform:Linux,a:a2~,b:a1~,x:b1,y:b3,back:b7,start:b6,guide:b23,leftshoulder:b11,rightshoulder:b10,leftx:a0,lefttrigger:b5,righttrigger:b4,dpup:h0.1,dpleft:h0.8,dpdown:h0.4,dpright:h0.2,"
```

As a replace for:
```
[Mappings]
0300f04c6d0400004fc2000011010000113349743="0300f04c6d0400004fc2000011010000113349743,Logitech G29 Driving Force Racing Wheel,platform:Linux,a:a2~,b:a1~,x:b1,y:b3,back:b7,start:b6,guide:b23,leftshoulder:b11,rightshoulder:b10,leftx:a0,lefttrigger:b5,righttrigger:b4,dpup:h0.1,dpleft:h0.8,dpdown:h0.4,dpright:h0.2,"
```
